README.md

```diff
@@ -1,123 +1,90 @@
-feedback
-========
+# Feedback
 
-This app handles user feedback related things.
+This is a public facing Ruby on Rails application that collects feedback from users in multiple ways (which are detailed below). This feedback information is then used to raise tickets in Zendesk.
 
-### Testing
+## Contact Page on GOV.UK
+![Contact Page on GOV.UK](/docs/screenshots/contact_page.png?raw=true "Contact Page on GOV.UK")
 
-To run unit tests, execute the following:
+## Contact Form on GOV.UK
+![Contact Form on GOV.UK](/docs/screenshots/contact_form.png?raw=true "Contact Form on GOV.UK")
 
-    bundle exec rake
+### Technical documentation
 
-### Testing with a mock `signon` strategy
+The app handles user feedback in the following ways:
 
-Launch using `bowl` from the `development` directory:
+1. Renders a contact page on GOV.UK: `https://www.gov.uk/contact/govuk`
+2. Collects feedback from other contact forms. Pages using the `static` app will have a link at the bottom of the page asking: `Is there anything wrong with this page?`. Clicking this link will show you a form. The data from the form will be submitted to the `feedback` app.
+3. Raises tickets based on feedback to Zendesk. The app does this by submitting data to the `support` app.
+3. A certain subset of tickets ('assisted digital') are stored twice: they're sent once to Zendesk and once to a Google spreadsheet via the Google API. More documentation about that here: [docs/assisted_digital_feedback.md](docs/assisted_digital_feedback.md).
+4. Handles email survey signups. A banner that shows up on `gov.uk` will ask you to provide an email to signup to a survey. `feedback` will then email you a link to the survey. However, it does not collect the answers. More documentation about that here: [docs/email_survey_signups.md](docs/email_survey_signups.md) 
 
-    development> bowl feedback
+### Dependencies
 
-### Testing with real authorisation
+- [alphagov/slimmer](https://github.com/alphagov/slimmer) - provides Rack middleware for applying a standard header and footer around pages returned by a Ruby (Rack) application
+- [alphagov/govuk_frontend_toolkit](https://github.com/alphagov/govuk_frontend_toolkit) - provides Sass and Javascript files for the application's frontend
+- [alphagov/gds-api-adapters](https://github.com/alphagov/gds-api-adapters) - provides a set of API adapters to work with the GDS APIs
+- [alphagov/govuk-content-schemas](https://github.com/alphagov/govuk-content-schemas) - provides schemas and examples of the content that uses them on GOV.UK
+- [alphagov/govuk-content-schema-test-helpers](https://github.com/alphagov/govuk-content-schema-test-helpers) - provides test helpers for working with [alphagov/govuk-content-schemas](https://github.com/alphagov/govuk-content-schemas)
+- [alphagov/govuk-lint](https://github.com/alphagov/govuk-lint) - configures various linters to comply with our style guides.
```

Comment:
Similar to my comment on alphagov/support#325 we should list the dependencies that can't be read out of the Gemfile; support-api, support, static, etc...

Reply:
I've added dependency descriptions for `support-api`, `support` and `static`. Not sure if there are others I should mention.